src/components/Calculator.js

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable */
```

Comment:
[OPTIONAL]

- Avoid `disabling eslint` feature instead, you can fix these by implementing best practices in your code.